### PR TITLE
[ADD][sale_order_contacts]Add more than one sale person and security task#2184

### PR DESCRIPTION
**Pull request by IsabelEsparza**
Create new many2many in the cintacts.
create new rule for show only the contacts assgned to the user.

### CODE REVIEW

**Comment by IsabelEsparza** [comment]
@alan196 Please check it!.

Thanks

File: addons/sale_order_contacts/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import logging
+from odoo import api, fields, models
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    empl_id = fields.Many2many(
+        'res.users', string='Sale persons')
+
+    # @api.multi
```
Comment (alan196) [inline addons/sale_order_contacts/models/res_partner.py]: Remove commented code.

File: addons/sale_order_contacts/models/res_partner.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2019, Jarsa Sistemas, S.A. de C.V.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+from odoo import api, fields, models
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    empl_id = fields.Many2many(
```
Comment (alan196) [inline addons/sale_order_contacts/models/res_partner.py]: please use a better field name also if it's a m2m field it must end with ids.

I think it's better seller_ids (check if the field exists or not in the model)

File: addons/sale_order_contacts/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2019, Jarsa Sistemas, S.A. de C.V.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Report: Purchase Order',
```
Comment (alan196) [inline addons/sale_order_contacts/__manifest__.py]: Why Report Purchase Order?

Please define the correct name

File: addons/sale_order_contacts/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    'summary': 'Custom report for purchase order',
+    'version': '11.0.1.0.0',
+    'category': 'Purchase',
```
Comment (alan196) [inline addons/sale_order_contacts/__manifest__.py]: Sale

File: addons/sale_order_contacts/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+    'author': 'Jarsa Sistemas',
+    'website': 'https://www.jarsa.com.mx',
+    'license': 'AGPL-3',
```
Comment (alan196) [inline addons/sale_order_contacts/__manifest__.py]: LGPL (also check the Licence in the top of every python file)

File: addons/sale_order_contacts/__manifest__.py [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+    'depends': [
+        'hr',
```
Comment (alan196) [inline addons/sale_order_contacts/__manifest__.py]: Why this module depends on hr??

**Review by alan196 — CHANGES_REQUESTED** [review]
I think this change can be applied in the instance module sysartec, as it's a small security change

**Pull request merged**